contrib/opencensus-ext-azure/examples/logs/correlated.py

```diff
@@ -14,7 +14,7 @@
 
 import logging
 
-from opencensus.ext.azure.log_exporter import AzureLogHandler
+from opencensus.ext.azure.logs_exporter import AzureLogHandler
```

Comment:
This is a breaking change, we need to:
1. Update the changelog to mention it, so users would know.
2. Bump the version when we do a release.

I would suggest that we don't touch this for now (since we haven't yet finalized whether to take plural form or not).

Reply:
You're right. I did not realize this. I will revert this for now.